rust/publish-ghost/src/lib.rs

```diff
@@ -482,16 +486,19 @@ impl Cli {
         };
 
         // Get document title and other metadata
-        let title = doc
-            .inspect(|root: &Node| {
+        let title = if self.title.is_none() {
+            doc.inspect(|root: &Node| {
                 if let Node::Article(article) = root {
                     if let Some(inlines) = &article.title {
                         return Some(codec_text::to_text(inlines));
                     }
                 }
                 None
             })
-            .await;
+            .await
+        }else{
```

Comment:
This will break our formatting checks on CI. Can you please run `make -C rust fix lint test` before pushing to deal with think like this.

```suggestion
        } else {
```